src/pages/zh/dev-tools/cli/bubblegum/create-cnft.md

```
@@ -0,0 +1,154 @@
+
+6. 在集合中创建：
+```bash
+mplx bg nft create my-tree \
+  --name "Collection Item #1" \
+  --image ./nft.png \
+  --collection 7kPqYxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+```
+
+## 输出
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)